test/e2e/edu-1990n-helpers.js

```diff
@@ -0,0 +1,14 @@
+function completeServicePeriods(client, data) {
```

Comment:
Could we put this in EduHelpers and make it a generic `completeToursOfDuty()` which the `completeServicePeriods()` in EduHelpers could call? Could loop over `data.toursOfDuty` and fill in all the data it has (so if we don't have `serviceStatus`, we pass it by).